### PR TITLE
github: add a page about pull requests

### PR DESCRIPTION
**Pull request by kopiczko**
Fixes https://github.com/giantswarm/giantswarm/issues/5620.

I'm open to suggestions.

### CODE REVIEW

File: github/pull_requests.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Pull Requests
+
+This page provides a guidelines on how to craft pull requests. The goal is to have semi-structure and checklist which make the pull request easier to review.
```
Comment (xh3b4sd) [inline github/pull_requests.md]:
```suggestion
This page provides guidelines on how to craft pull requests. The goal is to have some semi-structure 
and checklist which make the pull request easier to review.
```

File: github/pull_requests.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## General Guidelines
+
+- Think about the reader while crafting a PR. Take time to split the work after prototyping.
+- Make the scope of the PR small.
```
Comment (xh3b4sd) [inline github/pull_requests.md]:
```suggestion
- Keep the scope of the PR small.
```

File: github/pull_requests.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- Try to change things in as few packages as possible. If changes are needed in too many packages at once it's an indication that the PR scope is too broad or package design is wrong.
```
Comment (xh3b4sd) [inline github/pull_requests.md]:
```suggestion
- Try to change things in as few packages as possible. If changes are needed in too many 
  packages at once it's an indication that the PR scope is too broad or package design is wrong.
```

**Review by xh3b4sd — APPROVED** [review]
I like this. Thank you very much. I would like to add to review your own PR once it got crafted so you yourself get the chance to spot issues you can address yourself which also makes the life of your colleagues easier.

File: github/pull_requests.md (hunk 3 above)
Comment (kopiczko) [reply github/pull_requests.md]: I decided to not break lines for practical reasons - it's easier to add suggestions w/o dealing with line length. I'd keep it how it is unless it's breaking your heart.

**Comment by kopiczko** [comment]
> I would like to add to review your own PR once it got crafted so you yourself get the chance to spot issues you can address yourself which also makes the life of your colleagues easier.

I like that. Can you provide a suggestion?

**Comment by xh3b4sd** [comment]
I pushed a commit.

File: github/pull_requests.md (hunk 3 above)
Comment (xh3b4sd) [reply github/pull_requests.md]: Disagree and commit.

**Review by MarcelMue — APPROVED** [review]
<3

**Pull request merged**